camera_error_handler.py

```diff
@@ -0,0 +1,34 @@
+"""
+Exception handler for the Raspicam 
+
+Used alongside the 'with' magic Python word.
+eg. with CameraException(self.client, self.backend_name)
+"""
+from json import dumps
+from traceback import format_exc
+
+from config import read_configs
+from topics import Camera
+
+class CameraException:
+    def __init__(self, client, backend:str):
```

Comment:
Just for consistency with other parts of this repo (and I think it's standard too) we should have a space after `:`